ndn_python_repo/clients/putfile.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, app: NDNApp, prefix: NonStrictName, repo_name: NonStrictName)
         self.repo_name = repo_name
         self.encoded_packets = {}
         self.pb = PubSub(self.app, self.prefix)
+        self.pb.base_prefix = self.prefix
```

Comment:
Oh, that's why I had the issue in my own client, but not in the `example/putfile` client, I just forgot to call `set_base_prefix` :see_no_evil: 

I would push towards removing the line from `example/putfile.py`. Then an API user can not forget to call that line anymore. Do you agree with that? I can add another commit changing `example/putfile.py`. This would resolve your comment, right?